test/e2e/run.sh

```diff
@@ -1231,6 +1233,7 @@ fi
 
 if [ "$reinstall_containerd" == "1" ] || [ "$reinstall_crio" == "1" ] || ! vm-command-q "( type -p containerd || type -p crio ) >/dev/null"; then
     vm-install-cri
+    is-hooked "on_cri_install" && run-hook "on_cri_install"
```

Comment:
nitpick 1: why to quote `on_cri_install` here ?

nitpick 2: I've always wondered why isn't the `is-hooked` test folded into `run-hook` ? Then we could always unconditionally go `run-hook foo` instead of the now idiomatic/repetitive `is-hooked foo && run-hook foo`...

Reply:
Quotations fixed. I agree with integrating is-hooked into run-hook. I do not remember anymore why-oh-why did I split it in two. It may have been a thought failure in thinking how to return correct exit status from a function.

I'd like to write another PR that removes calling is-hooked before run-hook.